doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+Learn more about our [bar charts](chart-types-bar.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Pie Chart
+
+The $PlatformShort$ Pie Chart, or Pie Graph, is a very common part-to-whole chart type. Part-to-whole charts show how categories (parts) of a data set add up to a total (whole) value. Categories are shown in proportion to other categories based on their value percentage to the total value being analyzed. A pie chart renders data values as sections in a circular, or pie-shaped graph. Each section, or pie slice, has an arc length proportional to its underlying data value. The total values represented by the pie slices represent a whole value, like 100 or 100%. Pie charts are perfect for small data sets and are easy to read at a quick glance.
+
+Learn more about our [pie charts](../pie-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Line Chart
+
+The $PlatformShort$ Line Chart, or Line Graph is a type of category line graph shows the continuous data values represented by points connected by straight line segments of one or more quantities over a period time for showing trends and performing comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) are showing a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart.
+
+Learn more about our [line charts](chart-types-line.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Doughnut Chart
+
+The $PlatformShort$ Doughnut Chart, or Doughnut Graph, is a variant of a Pie Chart, proportionally illustrating the occurrences of a variable in a circle to represents parts of a whole. The doughnut chart has a circular opening at the center of the pie chart, where a title or category explanation can be displayed. Doughnut charts can support multiple concentric rings, with built-in support for visualizing hierarchical data.
+
+Learn more about our [doughnut charts](../doughnut-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Area Chart
+
+Learn more about our [area charts](chart-types-area.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/fa93a0a6-c870-4734-9717-5fa353445312?mode=build&sha=190f2102618cff2469e36d413cf34c7a759af170

The dat is same as in the Line Chart:
[Line Chart Data.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5674119/Line.Chart.Data.xlsx)

Reply:
Here if we want to give a different example we can use the Stacked Spline Area Chart (in the middle) from here:

https://share.goabstract.com/daa3712e-a050-40ee-8197-01b904d5a297?mode=build&sha=cbb507583316514fd9d47f72830350341a36dbc0